sdk/identity/identity/README.md

```diff
@@ -59,6 +59,8 @@ For systems without a default web browser, the `az login` command will use the d
 
 To authenticate Azure SDKs within web browsers, we currently offer the `InteractiveBrowserCredential`, which can be set to use redirection or popups to complete the authentication flow. It is necessary to [create an Azure App Registration](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration) in the portal for your web application first.
```

Comment:
It might be useful to call out the differences in the app registration between `auth-code` and `implicit-grant`. While the doc you have linked here talks about it, it may not always be obvious for people who are moving from implicit to auth-code flow.

Reply:
On it!

Reply:
I've added more information! Please review again 🌞